### ISSUE TITLE
Thonny can't find StreamDeck Package

### ISSUE THREAD

**Issue by sproehm** [abcminiuser/python-elgato-streamdeck#34]
Hi,

Thanks so much for this library.   I am having difficulty getting it to be recognized by Thonny.   I am running Raspbian GNU/Linux 10 (buster).   I followed the raspberry pi installation instructions (including the pip3 install streamdeck).   Everything installed correctly and without errors.   I also installed the package from within Thonny (Tools->Manage Packages).  I did reboot the raspberry pi (Rpi 4B) after the install.   When trying to run the example python app, I am getting the following error:
Traceback (most recent call last):
  File "/home/pi/Desktop/StreamDeck/StreamDeck.py", line 16, in <module>
    from StreamDeck.DeviceManager import DeviceManager
  File "/home/pi/Desktop/StreamDeck/StreamDeck.py", line 16, in <module>
    from StreamDeck.DeviceManager import DeviceManager
ModuleNotFoundError: No module named 'StreamDeck.DeviceManager'; 'StreamDeck' is not a package

Any help would be greatly appreciated!

**Comment by abcminiuser** [comment]
Archiving my email response here, for future searchers:

```HID access via Python is currently a real mess. The TL;DR answer if you don't want to read the below is to try running a `sudo apt-get remove libhidapi-hidraw0` on the Raspberry Pi.

The longer answer is that there are two packages that I support for the HID communication backend to the StreamDeck: `hid` and `hidapi`. The `hid` library is the better of the two, as it just wraps the system installed HIDAPI library, which means you can keep the actual underlying OS code up to date just by making sure the system packages are up to date. The `hidapi` library is self-contained which makes it easier to install on Windows, but contains a whole bunch of bugs which can cause various strange issues if the library is used over a long period of time.

Further complicating matters is that the `hid` and `hidapi` packages conflict with each other, so you can only install one or the other. The library is probing for both and your log shows it found the `hid` library, which is good! The probe is failing however due to the last complication: there's two possible OS layers the `hid` package can use, and it will automatically pick the first one it finds.

Firstly is libhidapi-hidraw, which works decently but cannot talk to the original StreamDeck due to an API limitation. Second is libhidapi-libusb, which works for all the StreamDecks. The problem: the `hid` Python library will use libhidapi-hidraw over libhidapi-libusb if both are installed, which isn't what we want. Removing the `libhidapi-hidraw0` should force the `hid` library to find the only working OS layer, the one we want.


To summarize: the possible library/system package combinations are:

    1) python-elgto-streamdeck (python library) -> hidapi (python library)                                                               [works, buggy on some systems]
    2) python-elgto-streamdeck (python library) -> hid (python library) -> libhidapi-libusb (OS package)           [works]

    3) python-elgto-streamdeck (python library) -> hid (python library) -> libhidapi-hidraw (OS package)          [fails due to a blacklist I added, as it cannot talk to original StreamDecks]


Hope that helps - if not please let me know and I'll debug further.```
